src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/Rules/BaseIndentationFormattingRule.cs

```diff
@@ -62,46 +65,47 @@ private void AddNextIndentBlockOperations(List<IndentBlockOperation> list, Synta
             _vbHelperFormattingRule.AddIndentBlockOperations(list, node, in nextOperation);
         }
 
-        private void AdjustIndentBlockOperation(List<IndentBlockOperation> list)
+        private void AdjustIndentBlockOperation(SegmentedList<IndentBlockOperation> list)
         {
-            list.RemoveOrTransformAll(
-                (operation, self) =>
+            using var _ = ArrayBuilder<IndentBlockOperation>.GetInstance(out var copy);
```

Comment:
💡 I would prefer to see the current `RemoveOrTransformAll` extension method rewritten to support `T : struct` (it's a really small change that allows in-place update)

Reply:
yes.  much nicer.